src/index.html

```diff
@@ -78,22 +78,39 @@ <h4><a href="https://pub.dartlang.org/">Dart packages</a></h4>
     <div class="row">
       <div class="col-sm-5 col-lg-4">
         <div class="content">
-          <h1>Core goals</h1>
+          <h1>Dart’s Core Tenets</h1>
```

Comment:
"Tenets" doesn't seem like a word most people know. What's wrong with "goals"?

Reply:
If you think "goals" implies something we might not have achieved, "principles" is more familiar than "tenets". But maybe we can be less lofty and just say something like "Why Dart?" or "What's special about Dart" or maybe just drop the heading entirely.

Reply:
I like the idea of using "principles" instead. But actually, "Why Dart?" is probably a better way to both get attention and also a good headline for what we are trying to articulate in this section.